GooglePlacesAutocomplete.js

```diff
@@ -682,9 +694,7 @@ export default class GooglePlacesAutocomplete extends Component {
   }
   render() {
     let {
-      onFocus,
       clearButtonMode,
-      ...userProps
```

Comment:
This is kinda important.  This passes all the props from the `textInputProps` prop to the `TextInput`.